partner_financial_risk/tests/test_partner_financial_risk.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+# © 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.account.tests.account_test_classes import AccountingTestCase
+from odoo import fields
+
+
+class TestPartnerFinancialRisk(AccountingTestCase):
```

Comment:
Don't depend on AccountingTestCase but directly on TransactionCase and this will work on Travis. Take into account that the test on Travis are not performed the same as in local: first the module is installed, and then Odoo is re-run executing the tests.